news/2 Fixes/10060.md

```diff
@@ -0,0 +1 @@
+Ensure IPyWidgets get loaded correctly when loading resources from the CDN, Remote Jupyter or local the Python Environment.
```

Comment:
Still requires some more testing, but this is ready for review.
Will look into adding more tests or at least re-enabling some of the widget tests tomorrow.